ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-postsubmits.yaml

```diff
@@ -54,3 +54,37 @@ postsubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
+  - agent: kubernetes
+    branches:
+    - ^master$
+    cluster: api.ci
+    decorate: true
+    name: branch-ci-open-cluster-management-registration-operator-master-publish
+    spec:
+      containers:
+      - args:
+        - pipeline-manifest/update
```

Comment:
/update is going to call /_update which calls /_add the first time (or /_update --> /_delete and /_add on subsuequent runs)... which will require `PIPELINE_MANIFEST_COMPONENT_SHA256`, which is the sha of the git commit.  Do we have that?